sql/catalyst/src/main/antlr3/org/apache/spark/sql/catalyst/parser/SparkSqlParser.g

```diff
@@ -2320,6 +2320,19 @@ regularBody[boolean topLevel]
    )
    |
    selectStatement[topLevel]
+   |
+   (LPAREN selectStatement[true]) => nestedSetOpSelectStatement[topLevel]
```

Comment:
Passing `true` or `false` here doesn't really matter. This `topLevel` argument works only in `setOpSelectStatement` rule. We will pass in the real `topLevel` argument into the rule later.